azclishell/az_completer.py

```diff
@@ -131,6 +131,7 @@ def get_completions(self, document, complete_event):
 
         text = reformat_cmd(text)
         if text.split():
+
```

Comment:
However, it doesn't look like it has addressed everything. e.g. https://github.com/Azure/azure-cli-shell/issues/9
@oakeyc please take a look.